recipes/scikit-image/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "0.12.3" %}
+
+package:
+  name: scikit-image
+  version: {{ version }}
+
+source:
+  fn: scikit-image-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/s/scikit-image/scikit-image-{{ version }}.tar.gz
+  md5: 04ea833383e0b6ad5f65da21292c25e1
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt  # [not osx]
```

Comment:
I don't have a strong feeling about this, but it feels weird to have the build commands split in two files. I would either eliminate this line as it is not needed. Or I would removed the `build.sh` and add a `export LDFLAGS="-L$PREFIX/lib -Wl,-rpath,$PREFIX/lib"  # [osx]` line here.